pymeasure/adapters/visa.py

```diff
@@ -21,7 +21,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #
-
+import typing
```

Comment:
There are no plans to support static typing in PyMeasure. This adds a lot of additional code to manage, without a clear advantage for this project. Type documentation should be obvious from the variable name or clearly stated in the docstring.

Reply:
>or clearly stated in the docstring.

docstrings mustn't contain types. It's much more better to store them as annotations.

>There are no plans to support static typing in PyMeasure.

There are no static typing in python. Anything can be stored in annotations, it's just a convention to store types there.

Reply:
"incorrect" is a pretty strong word for what is basically a preference, don't you think?

Reply:
I should have been clear to say "type hinting" instead of "static typing", but the point is the same.

@KOLANICH, your comments on this project so far have taken a very condescending tone. If you have technical issues, you are welcome to raise them in a respectful way -- but you should recognize that your assessment may not align with the project goals. Please don't use "IMHO" as a substitute for being humble and considerate.

Reply:
Are there any real reasons not to use type hints instead of annotations in comments given that we don't support python 2? Type hints seem to be strictly better than annotations in comments.